### PR TITLE
LP-2675 ADG technical documentation for user_authn_override and utils apps

### PR DESCRIPTION
**Pull request by annas-usman**
[Ticket](https://philanthropyu.atlassian.net/browse/LP-2675)

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1706/workflows/workflows_id/jobs/1706/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (70.6%): Missing lines 53,67,80,94,116
openedx/adg/common/lib/mandrill_client/helpers.py (50.0%): Missing lines 41-46,48-49
openedx/adg/common/lib/mandrill_client/tasks.py (70.0%): Missing lines 44,56-57
openedx/adg/lms/webinars/admin.py (72.7%): Missing lines 81,155,158
openedx/adg/lms/webinars/constants.py (100%)
openedx/adg/lms/webinars/helpers.py (98.0%): Missing lines 124
openedx/adg/lms/webinars/managers.py (100%)
openedx/adg/lms/webinars/models.py (93.9%): Missing lines 165,211
openedx/adg/lms/webinars/tasks.py (100%)
openedx/adg/lms/webinars/tests/conftest.py (100%)
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/tests/test_tasks.py (100%)
openedx/adg/lms/webinars/tests/test_views.py (100%)
openedx/adg/lms/webinars/urls.py (100%)
openedx/adg/lms/webinars/views.py (100%)
-------------
Total:   234 lines
Missing: 22 lines
Coverage: 90%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1707/workflows/workflows_id/jobs/1707/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (70.6%): Missing lines 53,67,80,94,116
openedx/adg/common/lib/mandrill_client/helpers.py (50.0%): Missing lines 41-46,48-49
openedx/adg/common/lib/mandrill_client/tasks.py (70.0%): Missing lines 44,56-57
openedx/adg/lms/webinars/admin.py (72.7%): Missing lines 81,155,158
openedx/adg/lms/webinars/constants.py (100%)
openedx/adg/lms/webinars/helpers.py (98.0%): Missing lines 124
openedx/adg/lms/webinars/managers.py (100%)
openedx/adg/lms/webinars/models.py (93.9%): Missing lines 165,211
openedx/adg/lms/webinars/tasks.py (100%)
openedx/adg/lms/webinars/tests/conftest.py (100%)
openedx/adg/lms/webinars/tests/constants.py (100%)
openedx/adg/lms/webinars/tests/test_helpers.py (100%)
openedx/adg/lms/webinars/tests/test_tasks.py (100%)
openedx/adg/lms/webinars/tests/test_views.py (100%)
openedx/adg/lms/webinars/urls.py (100%)
openedx/adg/lms/webinars/views.py (100%)
-------------
Total:   234 lines
Missing: 22 lines
Coverage: 90%
-------------
```

File: openedx/adg/lms/utils/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+"""
+`utils` app consists of utility methods being used throughout the platform, whether helping methods used in making model
```
Comment (ahmed-faraz46) [inline openedx/adg/lms/utils/__init__.py:2]: *consists of **ADG** utility methods

File: openedx/adg/lms/user_authn_override/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+"""
+`user_authn_override` app overrides the existing authentication forms by creating and returning new forms with the
```
Comment (ahmed-faraz46) [inline openedx/adg/lms/user_authn_override/__init__.py:2]: Basically the app is written to override any kind of implementation of user_authn core app and as an example we can mention the we have overrode forms

**Review by ahmed-faraz46 — CHANGES_REQUESTED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1736/workflows/workflows_id/jobs/1736/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Comment by annas-usman** [comment]
quality action is failing because of pylint. Even though pylint is not raising any error but as it is also not producing a report which is the basis of displaying error in the action file therefore it fails. Hence we need to update the action file as done [here](https://github.com/philanthropy-u/edx-platform/pull/1924/files) in PhilU, for ADG as well

**Review by asadali145 — APPROVED** [review]

**Review by ahmed-faraz46 — APPROVED** [review]

**Pull request merged**